src/styles/typography.ts

```diff
@@ -0,0 +1,16 @@
+import { TextStyle } from 'react-native'
+
+export const fontByWeight = {
+  heavy: 'AvenirNextLTPro-Heavy',
+  bold: 'AvenirNextLTPro-Bold',
+  demiBold: 'AvenirNextLTPro-DemiBold',
+  medium: 'AvenirNextLTPro-Medium',
+  regular: 'AvenirNextLTPro-Regular',
+  light: 'AvenirNextLTPro-Light',
+  thin: 'AvenirNextLTPro-Thin',
+  ultraLight: 'AvenirNextLTPro-UltLt'
+}
+
+export const fontWeight = (weight: keyof typeof fontByWeight): TextStyle => ({
```

Comment:
Thinking a name like `fontFromWeight` or something would be a bit more clear here

Reply:
What about just `font`? I feel like we could add things like letter spacing to this too

Reply:
Yea `font` or `fontStyle` or something like that works for me. I felt that fontWeight was a bit less clear